cli/angular/README.md

```diff
@@ -0,0 +1,154 @@
+> A little helper to unit test React components in the open source [Cypress.io](https://www.cypress.io/) test runner **v7.0.0+**
```

Comment:
We normally gitignore the `cli/*` packages, and don't commit them, since they are generated at build time. You can remove these and add them to `gitignore`.

Reply:
Good catch!